specials/SPC18_Matias_SpiesVsMercs.Altis/initServer.sqf

```diff
@@ -0,0 +1,10 @@
+#include "do_not_edit\script_component.hpp"
+#include "do_not_edit\initServer.sqf"
+
+// {tag}_fnc_{name}
+
+mission_fnc_victoryHandler = compile preprocessFileLineNumbers "victory_handler.sqf";
+
+mission_victoryHandlerPFH = [{
+    call mission_fnc_victoryHandler;
+}, 3] call CBA_fnc_addPerFrameHandler;
```

Comment:
```suggestion
[{ CBA_missionTime > 5 }, {
    mission_victoryHandlerPFH = [{
        call mission_fnc_victoryHandler;
    }, 3] call CBA_fnc_addPerFrameHandler;
}] call CBA_fnc_waitUntilAndExecute;
```

We want to wait until the mission has actually been running, to prevent it from ending right at the start before any players have loaded in.

Reply:
Done